### DISCUSSION TITLE
Installation without internet

### DISCUSSION THREAD

**Discussion by antonio-tg** [ohmyzsh/ohmyzsh#12162] [Help]
I have to work with a server without internet access and I would love to be able to install oh-my-zsh, could you help me with that please, all I can do on the server is upload files.

**Comment by mcornella** [comment]
You can download the latest zip file (see screenshot) and then follow the steps in [[1]] to extract it somewhere (instead of doing the `git clone`) and apply the .zshrc file.

Note that automatic upgrades will not work, so also make sure to disable them by adding the setting documented in [[2]] (this should go in the .zshrc file as well, before OMZ is sourced).

[1]: https://github.com/ohmyzsh/ohmyzsh#manual-installation

[2]: https://github.com/ohmyzsh/ohmyzsh/wiki/Settings#omzupdate-mode

![Captura desde 2024-01-16 19-46-17](https://github.com/ohmyzsh/ohmyzsh/assets/1441704/80041ac8-1e3d-4386-9212-49701baafa87)

**Reply by antonio-tg** [reply]
Thank you very much, it turned out beautiful.
